src/auth0/logoutButton.js

```diff
@@ -0,0 +1,14 @@
+import { useAuth0 } from '@auth0/auth0-react';
+
+const LogoutButton = () => {
+  const { logout } = useAuth0();
+
+  return (
+    <button type="button" className="all-btns btn bm-burger-bars text-white" onClick={() => logout({ returnTo: window.location.origin })}>
```

Comment:
I would recommend we use react DOM router's functions to redirect/navigate our user to a different page after logout instead of manipulating the DOM directly with `window.location`